src/model/Model.java

```diff
@@ -296,8 +297,10 @@ public void updateCachedIssue(TurboIssue issue){
 		TurboIssue tIssue = getIssueWithId(issue.getId());
 		if(tIssue != null){
 			tIssue.copyValues(issue);
-		}else{
+			logger.info("Updated issue: " + issue.getId());
```

Comment:
This kind of detailed logs can be done at `debug` level. Furthermore, try to record what exactly was changed (i.e. the new values). Can be helpful in debugging cases of 'missing labels' etc.